hair_salon/data/base_automation.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+    <record id="break_booking_automation" model="base.automation">
+        <field name="name">Delte Main Event</field> 
+        <field name="model_id" ref="calendar.model_calendar_event"/>
+        <field name="action_server_ids" eval="[(6, 0, [ref('break_booking_server_action')])]"/>
+        <field name="trigger">on_unlink</field>
+        <field name="filter_domain">[("appointment_type_id", "!=", False)]</field>
+    </record>
+    <record id="delete_subpart_calendar_event" model="base.automation">
```

Comment:
I wonder if this whole logic wouldn't benefit from adding an `on_delete` strategy on the new m2o fields that you create (like cascade)

Reply:
I'll need more explaination.